src/txValidators.ts

```diff
@@ -157,18 +161,26 @@ function *validateStakeCredentials(certificates: Certificate[] | undefined, with
 
 const validateMint = (mint: Mint) => validateMultiasset(mint, validateInt64, 'transaction_body.mint')
 
-function *validateCollaterals(txCollaterals: Collateral[]): ValidatorReturnType {
-    yield* validateListConstraints(txCollaterals, 'transaction_body.collaterals', false)
+function *validateCollateralInputs(txCollateralInputs: CollateralInput[]): ValidatorReturnType {
+    yield* validateListConstraints(txCollateralInputs, 'transaction_body.collateral_inputs', false)
 
-    for (const [i, collateral] of txCollaterals.entries()) {
-        yield* validateUint64(collateral.index, `transaction_body.collaterals[${i}].index`)
+    for (const [i, collateralInput] of txCollateralInputs.entries()) {
+        yield* validateUint64(collateralInput.index, `transaction_body.collateral_inputs[${i}].index`)
     }
 }
 
 function *validateRequiredSigners(txRequiredSigners: RequiredSigner[]): ValidatorReturnType {
     yield* validateListConstraints(txRequiredSigners, 'transaction_body.required_signers', false)
 }
 
+function *validateTxReferenceInputs(txReferenceInputs: TransactionInput[]): ValidatorReturnType {
```

Comment:
Not sure if we need "tx" in the names.

Reply:
ISTM that the `tx` usage is rather inconsistent on several places (e.g. a few lines above, we have `validateRequiredSigners(txRequiredSigners ...)`). I will fix this.

Reply:
Renamed in https://github.com/vacuumlabs/cardano-hw-interop-lib/pull/7/commits/86bea77728affecd2267f3c62706f5a15c07d748. I dropped `tx` from all items except `txInput` and `txOutput` (so that we avoid the general words `input` and `output` as an argument; as it is done elsewhere, too).